File: 01_ProgrammingBasics/_02_ConditionalStatements/_01_ExcellentResult.java
package _01_ProgrammingBasics._02_ConditionalStatements;

import java.util.Scanner;

public class _01_ExcellentResult {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read the grade input
        int grade = Integer.parseInt(scan.nextLine());

        // Determine if the grade is excellent
        boolean isExcellent = grade >= 5;

        // Print "Excellent!" if the grade is excellent
        if (isExcellent) {
            System.out.println("Excellent!");
        }

        scan.close();
    }
}

File: 23_C#Basics/_02_ConditionalStatements/_01_ExcellentResult/_01_ExcellentResult.cs
using System;

public class _01_ExcellentResult
{
    public static void Main(string[] args)
    {
        // Read the grade input
        double grade = double.Parse(Console.ReadLine());

        // Determine if the grade is excellent
        bool isExcellent = grade >= 5.5;

        // Print "Excellent!" if the grade is excellent
        if (isExcellent)
        {
            Console.WriteLine("Excellent!");
        }
    }
}
